sklearn/linear_model/_logistic.py

```diff
@@ -1763,11 +1783,22 @@ def fit(self, X, y, sample_weight=None):
             Array of weights that are assigned to individual samples.
             If not provided, then each sample is given unit weight.
 
+        **params : dict
+            Parameters to pass to the underlying splitter and scorer.
+
+            .. versionadded:: 1.4
+
         Returns
         -------
         self : object
             Fitted LogisticRegressionCV estimator.
         """
+        if params and not _routing_enabled():
+            raise ValueError(
+                "params is only supported if enable_metadata_routing=True."
+                " See the User Guide for more information."
```

Comment:
Would be useful to get the link to the user tag.

Reply:
I mean this page: https://scikit-learn.org/stable/metadata_routing.html

Reply:
I think it will definitely be useful. But if we add it here in one place whereas there are a number of other cases like this (this particular ValueError) that don't contain this link, won't that be a bit inconsistent?

Reply:
We can open a subsequent PR to improve the consistency.